doc/io/influxdb/loader.md

```diff
@@ -66,5 +66,26 @@ ctk load table \
   --cratedb-sqlalchemy-url="crate://admin:dZ...6LqB@green-shaak-ti.eks1.eu-west-1.aws.cratedb.net:4200/testdrive/demo?ssl=true"
 ```
 
+## Parameters
+
+### `if-exists`
+
+The target table will be created automatically, if it does not exist. The
+`if-exists` URL query parameter can be used to configure this behavior if it
+does. The default value is `fail`, the possible values are:
```

Comment:
```suggestion
The target table will be created automatically, if it does not exist. If it
does exist, the `if-exists` URL query parameter can be used to configure this
behavior. The default value is `fail`, the possible values are:
```

Wasn't clear to me with the "if it does" at the end of the sentence what it referred to, had to read it a few times until I got the reference.